package.json

```diff
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/AgileVentures/LocalSupport",
   "dependencies": {
-    "bower": "^1.3.12"
+    "bower": "^1.3.12",
```

Comment:
Hi @nisevi , what do you think about this change? Does it solve for our current problem?

Reply:
@faithngetich nice work :+1: